src/domain_factory/price/discount_rate_factory.py

```diff
@@ -0,0 +1,28 @@
+from domain_factory.price.price_factory import PriceFactory
+
+
+class InterestRateFactory(PriceFactory):
```

Comment:
I think this class value and the comments needs to be updated from Interest to Discount

Reply:
done